python/ray/serve/replica.py

```diff
@@ -295,6 +299,9 @@ def __init__(
                 )
             )
 
+    def check_health(self):
```

Comment:
This needs to be an `async` function.

Reply:
Thanks, I was in the car and not willing to recompile ray to run tests :D